source/consumers/elastic/media-insights-elasticsearch.yaml

```diff
@@ -42,6 +39,9 @@ Parameters:
   AnalyticsStreamArn:
     Description: "Arn of the MIE kinesis data stream"
     Type: String
+  MIEAdminRoleArn:
```

Comment:
This is the role we will pass to the downstream analytics stack for use in the elasticsearch access policy.